### PR TITLE
Added english language

### PR DESCRIPTION
**Pull request by AmirHosein-Gharaati**
Added English language

### CODE REVIEW

File: main.dart [hunk 1]
```diff
@@ -0,0 +1,362 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
```
Comment (AmirAG80) [inline main.dart:2]: test

File: main.dart [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+
+class _HomePageState extends State<HomePage> {
+
+
+  String language = 'fa';
+  Map strings = {
+    'fa':{'title':'ثبت یادداشت های مهم شما در نوتیفیکیشن :)','main':'متن یادداشت','button':'ثبت یادداشت','switch':'نمایش دادن نوتیفیکیشن'},
+    'eng':{'title':'(: Recording your important notes in notifications ','main':'Main Text','button':'Record Note','switch':'Show the Notification'},
+
+  };
```
Comment (AmirHosein-Gharaati) [inline main.dart:75]: A map for changing language

File: main.dart [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+        IconButton(
+          icon: Icon(
+            Icons.language,
+          ),
+          onPressed: (){
+            setState(() {
+              language = (language == 'fa') ? 'eng' : 'fa';
+            });
+          },
+        ),
```
Comment (AmirHosein-Gharaati) [inline main.dart:246]: added language icon

File: main.dart [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+          fontSize: 30.0,
+        ),
+        textAlign: TextAlign.center,
+      ),
+    );
+
+    final descriptionText = Padding(
+      padding: const EdgeInsets.symmetric(vertical: 8.0, horizontal: 12.0),
+      child: Text(
+        strings[language]['title'],
```
Comment (AmirHosein-Gharaati) [inline main.dart:164]: Texts like this changed from string to variable

**Comment by DeMonkeyCoder** [comment]
Thanks for your PR.

Using the manaul way is not a good option.
Use flutter internationalization tools:
https://flutter.dev/docs/development/accessibility-and-localization/internationalization

Then open a new pull request and also write fixes #14 in your PR's description

**Pull request closed without merging**